configure.ac

```
@@ -6,6 +6,8 @@ AC_CONFIG_AUX_DIR([build-aux])
 
 AC_REQUIRE_AUX_FILE([tap-driver.sh])
 
+CXXFLAGS="$CXXFLAGS -std=c++11"
```

I don't like this, because it will cause -std= downgrade on compilers that have default -std= greater than c++11. This patch causes fails on OpenSUSE Tumbleweed because of this.

---

Replacing with 

```suggestion
AX_CXX_COMPILE_STDCXX_11([], mandatory)
```

resolves the problem, at the expense of adding `autoconf-archive` as an autotools dependency. Not sure how you feel about this, as the error message when you don't have `autoconf-archive` is frustratingly bad. It took a lot of wasted google time to figure out that the problem wasn't in my syntax, but in the missing package. I can see many people being bitten by this and being at a loss for how to fix it.